Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabIntegrationManager.cpp

```diff
@@ -720,6 +720,12 @@ namespace AzToolsFramework
                         m_prefabOverridePublicInterface->AreOverridesPresent(selectedEntity))
                     {
                         QAction* revertAction = menu->addAction(QObject::tr("Revert Overrides"));
+                        revertAction->setToolTip(QObject::tr("Revert all overrides on this entity to whatever is in the prefab file."));
+                        if (m_prefabOverridePublicInterface->GetOverrideType(selectedEntity) == OverrideType::AddEntity)
+                        {
+                            revertAction->setToolTip(QObject::tr("Cannot revert overrides on entities that are added as overrides."));
+                            revertAction->setEnabled(false);
```

Comment:
I think it's better to not show this at all if this was an entity added as an override. This would be consistent with what we have been doing with other actions that user should not be able to perform. We simply hide them instead of showing them in a disabled state.

Reply:
A user not seeing a revert option on one entity and seeing it on others would rightly be confused as to why it exists on one and not on the other. @monroegm any thoughts ?

Reply:
I prefer making it disabled rather than hidden. As @raktalen points out, I think the user would be confused bc unlike other workflows there is a "+" icon on the entity. Additionally, it gives us a chance to let users know in tooltip that the entity can be deleted as @michabr mentions.